src/metabase/models/database.clj

```diff
@@ -34,23 +34,67 @@
 
 (models/defmodel Database :metabase_database)
 
+
+(defn- schedule-tasks!
+  "(Re)schedule sync operation tasks for DATABASE. (Existing scheduled tasks will be deleted first.)"
+  [database]
+  (try
+    (require 'metabase.task.sync-databases)
+    ((resolve 'metabase.task.sync-databases/schedule-tasks-for-db!) database)
+    (catch Throwable e
+      (log/error "Error scheduling tasks for DB:" (.getMessage e) "\n")
```

Comment:
Yeah, sorry, the error vs. errorf is probably a distraction. My intent was more around including the exception as the first argument to log/error (or log/errorf). Reading this, it seemed like most users wouldn't have debug logging enabled and so the error message might just be frustrating as you'd have to lower your log level and trigger the error again to get a stacktrace.